backend/Dockerfile.prod

```diff
@@ -10,4 +10,4 @@ RUN poetry install
 
 EXPOSE 80
 
-CMD ["poetry", "run", "backend"]
\ No newline at end of file
+CMD exec poetry run uvicorn zeno_backend.server:get_server --reload --host 0.0.0.0 --port 80
```

Comment:
Running it as a module didn't create an event loop in asyncio oddly. Also matches the dev setup.